HL2Collab/README.md

```diff
@@ -1,21 +1,61 @@
-# HL2Collab
-## _Multiplayer Networking on the HoloLens_
 ---
-# Instructions
+page_type: sample
+name: HoloLens Multiplayer Networking in Unity
+description: This sample shows how to network multiple HoloLens 2 devices using the built-in Unreal Engine networking system.
+languages:
+- cpp
+products:
+- windows-mixed-reality
+- hololens
+---
+
+# HL2Collab - HoloLens Multiplayer Networking in Unity
+
+![License](https://img.shields.io/badge/license-MIT-green.svg)
+
+Supported Unreal versions | Supported device | XR configuration
+:-----------------: | :----------------: | :----------------------:
+Unreal Engine 4.25+ | HoloLens 2 | OpenXR
+
+This sample shows how to network multiple HoloLens 2 devices together using Unreal Engine's built-in networking system. When using the sample, you'll need one instance of the application running as a Listen server. The Listen server is generally run on a desktop. Your client instances of the application can be other HoloLens 2 devices or another desktop.
+
+## Contents
+
+| File/folder | Description |
+|-------------|-------------|
+| `Config` | HoloLens and default configuration files. |
+| `Content` | Project files and assets. |
+| `Source` | Mixed Reality Toolkit for Unreal UX Tools plugin components. |
+| `images` | Image assets for the README. |
+| `.gitignore` | Define what to ignore at commit time. |
+| `HL2Collab.sln` | Project solution file. |
+| `HL2Collab.uproject` | Unreal Engine project file. |
+| `README.md` | This README file. |
+
+## Prerequisites
 
-- Open the *Level* blueprint in MainMenuMap. This contains instructions as well.
-- Set the **IP_TextField** variable to your computer's IP Adress
-- Create a Desktop Build. This will be the host
-- Create HL2 or more Desktop builds to act as clients
-- Run the Desktop version, and press "H" to begin hosting the map.
-- Run the HL2 clients, and "right pinch" to join the session to the host on the IP Address provided earlier
-- (Optional) Run any other Desktop executables, and press "J" to join
+<!-- Need these -->
+Needs OpenXR - need input from Sajid, Jackson, Summer
```

Comment:
In the Microsoft OpenXR Plugin GitHub, we have a section on adding the plugin to your project from the Unreal Marketplace:
https://github.com/microsoft/Microsoft-OpenXR-Unreal#install-the-plugin-from-the-unreal-marketplace

Once the Microsoft OpenXR Plugin is enabled, all other OpenXR prerequisite plugins will also be enabled, then setup for a HoloLens app resumes as normal